app/services/send_welcome_email_service.rb

```diff
@@ -1,12 +1,16 @@
 class SendWelcomeEmailService
-  def initialize(mailer:)
-    @mailer = mailer
+  class << self
+    def call(current_user:)
+      new.call(current_user: current_user)
+    end
   end
 
-  def execute(current_user:)
+  def call(current_user:)
     return if current_user.welcome_email_date_utc
 
-    @mailer.send_welcome_email(first_name: current_user.first_name, email: current_user.email).deliver_now
+    WelcomeEmailMailer.
+      send_welcome_email(first_name: current_user.first_name, email: current_user.email).
+      deliver_later(queue: "mailer")
```

Comment:
This might seem like a nitpick but please put the dot operator at the beginning of your lines. It might be easy to misread these as being calls to local methods.

Reply:
I've always put the dot at the end - https://github.com/rubocop-hq/ruby-style-guide#multi-line-method-chains

Happy to change if that's the team convention.

What do other's think?

Reply:
I prefer a leading dot. I'm surprised rubocop doesn't have an rule 🤔

Reply:
No prob - will update. Perhaps there is a way to get Rubocop to check for this ...

Reply:
Looks like govuk rubocop have disabled the dot check - https://github.com/alphagov/rubocop-govuk/search?q=dot&unscoped_q=dot

Reply:
@dankmitchell @faissalMT - given the above can I suggest that this PR is merged then we add the Rubocop dot check (and any fixes) in a subsequent PR? I wonder if we need a team vote on this also. Personally I don't care if we do a leading or trailing dot but I think this may be a contentious issue for others ... Maybe that's why GovUK disabled the check in the first place